en/mkdocs.yml

```diff
@@ -343,6 +343,13 @@ nav:
         - 'WSO2-2016-0168': 'security-announcements/security-advisories/2016/WSO2-2016-0168.md'
         - 'WSO2-2016-0169': 'security-announcements/security-advisories/2016/WSO2-2016-0169.md'
         - 'WSO2-2016-0170': 'security-announcements/security-advisories/2016/WSO2-2016-0170.md'
+    - 'Cloud Security Bulletin':
+      - '': 'security-announcements/cloud-security-bulletins/index.md'
+      - 'Asgardeo':
+        - '': 'security-announcements/cloud-security-bulletins/asgardeo/index.md'
+        - '2025':
+          - '': 'security-announcements/cloud-security-bulletins/asgardeo/2025/index.md'
+          - 'Asgardeo Security Bulletin - 2025 H1': 'security-announcements/cloud-security-bulletins/asgardeo/2025/asgardeo-2025-h1.md'
```

Comment:
```suggestion
          - 'Asgardeo Security Bulletin - 2025 H1': 'security-announcements/cloud-security-bulletins/asgardeo/2025/Asgardeo-2025-H1.md'
```

Can't recall whether the case of the file matters here

Reply:
fixed with [254be56](https://github.com/wso2/docs-security/pull/121/commits/254be56a29a5b22f1021bc9b0857348f7e08d81e)